cypress/integration/login.spec.ts

```diff
@@ -18,6 +18,16 @@ describe('Login process', () => {
         cy.get('[data-test="username"]').should('be.visible');
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
+
+    it('clicking on user avatar takes you to My Page', () => {
```

Comment:
You should be able to read this line as a sentence, including the `it` in the function name. A suitable description might be "(it) links the user avatar to My Page" or something like that.

This is a minor thing, and not something that needs to be fixed for this to be merged.

Reply:
aaah! got it!